addons/mail/static/src/owl/components/composer_text_input/composer_text_input.scss

```
@@ -15,17 +15,40 @@
     overflow: auto
 }
 
-.o_ComposerTextInput_mentionMenuItem {
-    white-space: nowrap;
-    overflow: hidden;
+.o_ComposerTextInput_mention_dropdown .o_mention_proposition {
 
-    &.o-mobile {
-        padding: 5px;
+    display: flex;
+    width: 100%;
+    padding: 3px 15px;
```

We are now using scss variables instead of hardcoded values. It makes it more consistent and easier to maintain and to theme (customize).

For the possible variables, see in file `/addons/web/static/lib/bootstrap/scss/_variables.scss`.
Whenever writing a css value, we should check in this file first if a variable exists for it (especially at the top of the file, or using search, or common sense).

For **padding** & **margin**: we need to use **$spacers**. For the possible values, see variable file above.

Summary

- 1->2px
- 2->4px
- 3->8px
- 4->12px
- 5->24px

but it depends on the base value (which is 1rem by default)

- 0px -> 0 (special case: I think it will not be rendered if it's 0 unless customized to be not 0, if we really need the 0 to appear eg. because of flex, then we need to still hardcode the 0 and not use the spacer)
Basically for 0 act with care and double check it works as expected.

Below some lines to check. I may have missed some, but the idea is to start adapting checking css like this. I'll probably do a pass over the rest of our existing code at some point.

```suggestion
    padding: map-get($spacers, 2) map-get($spacers, 4);
```

---

Done